apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/PooledSessionTests.cs

```diff
@@ -35,6 +35,9 @@ public class PooledSessionTests
         private static readonly SessionName s_sampleSessionName = new SessionName("project", "instance", "database", "session");
         // In various tests we don't care whether things are evicted "on" or "after" a particular tick, so add one tick.
         private static readonly TimeSpan OneTick = TimeSpan.FromTicks(1);
+        private static readonly TransactionOptions s_readWriteOptions = new TransactionOptions { ReadWrite = new Types.ReadWrite() };
```

Comment:
Semi-nit, but worth bearing in mind for the future - oneof cases like this can use target-typed new effectively:

```csharp
new TransactionOptions { ReadWrite = new() };
```